src/firebase-namespace.ts

```diff
@@ -287,10 +287,76 @@ const firebaseCredential = {
   },
 };
 
+export const credential = firebaseCredential;
+// const internal = new FirebaseNamespaceInternals();
+// export internal;
+export const internal = new FirebaseNamespaceInternals();
+
+export const SDK_VERSION = '<XXX_SDK_VERSION_XXX>';
+
+/**
+   * Initializes the FirebaseApp instance.
+   *
+   * @param {FirebaseAppOptions} [options] Optional options for the FirebaseApp instance.
+   *   If none present will try to initialize from the FIREBASE_CONFIG environment variable.
+   *   If the environment variable contains a string that starts with '{' it will be parsed as JSON,
+   *   otherwise it will be assumed to be pointing to a file.
+   * @param {string} [appName] Optional name of the FirebaseApp instance.
+   *
+   * @return {FirebaseApp} A new FirebaseApp instance.
+   */
+export function initializeApp(options?: FirebaseAppOptions, appName?: string): FirebaseApp {
+  return internal.initializeApp(options, appName);
+}
+
+/**
+ * Returns the FirebaseApp instance with the provided name (or the default FirebaseApp instance
+ * if no name is provided).
+ *
+ * @param {string} [appName] Optional name of the FirebaseApp instance to return.
+ * @return {FirebaseApp} The FirebaseApp instance which has the provided name.
+ */
+export function app(appName?: string): FirebaseApp {
+  return internal.app(appName);
+}
+
+/*
+ * Returns an array of all the non-deleted FirebaseApp instances.
+ *
+ * @return {Array<FirebaseApp>} An array of all the non-deleted FirebaseApp instances
+ */
+export function apps(): FirebaseApp[] {
+  return internal.apps;
+}
+
+export function cert(serviceAccountPathOrObject: string | object, httpAgent?: Agent): Credential {
+  const stringifiedServiceAccount = JSON.stringify(serviceAccountPathOrObject);
+  if (!(stringifiedServiceAccount in globalCertCreds)) {
+    globalCertCreds[stringifiedServiceAccount] = new ServiceAccountCredential(serviceAccountPathOrObject, httpAgent);
+  }
+  return globalCertCreds[stringifiedServiceAccount];
+}
+export function refreshToken(refreshTokenPathOrObject: string | object, httpAgent?: Agent): Credential {
+  const stringifiedRefreshToken = JSON.stringify(refreshTokenPathOrObject);
+  if (!(stringifiedRefreshToken in globalRefreshTokenCreds)) {
+    globalRefreshTokenCreds[stringifiedRefreshToken] = new RefreshTokenCredential(
+      refreshTokenPathOrObject, httpAgent);
+  }
+  return globalRefreshTokenCreds[stringifiedRefreshToken];
+}
+
+export function applicationDefault(httpAgent?: Agent): Credential {
+  if (typeof globalAppDefaultCred === 'undefined') {
+    globalAppDefaultCred = getApplicationDefault(httpAgent);
+  }
+  return globalAppDefaultCred;
+}
 
 /**
  * Global Firebase context object.
  */
+// TODO: remove?
+
```

Comment:
All contents of the `FirebaseNamespace` were removed so that they could be imported via the barrel export.